Symfony/demo/serverless.yml

```diff
@@ -0,0 +1,44 @@
+service: symfony
+
+provider:
+    name: aws
+    region: us-east-1
+    stage: dev
+    runtime: provided.al2
```

Comment:
Can you switch to the new runtime notation of Bref v2? https://bref.sh/news/02-bref-2.0#simpler-runtime-configuration